pkgs/by-name/fr/freecad/freecad-utils.nix

```diff
@@ -0,0 +1,57 @@
+{
+  runCommand,
+  buildEnv,
+  makeWrapper,
+  lib,
+}:
+rec {
```

Comment:
```suggestion
{
```

Remove `rec`.

Reply:
No, the `makeCustomizable` function used inside its own (`override`, `overrideAttrs` and `customize` methods of its result)

Reply:
Couldn't this be a recursive variable in a `let`?

Reply:
Maybe. Gonna try. But can you explain why `let-in` is better? Did not hear that `rec` is anti-pattern.

Reply:
Done